cookbooks/aws-parallelcluster-config/recipes/directory_service.rb

```diff
@@ -101,6 +101,29 @@
         ]
       )
     end
+
+    # Create directory for tools related to the directory service
+    directory_service_scripts_path = "#{node['cluster']['scripts_dir']}/directory_service"
```

Comment:
Should we create this folder at install time?

Reply:
In theory, yes; in practice, I think we should not.

Currently, we do not have any install-time recipe for directory-service. It looks a bit ver-killing to introduce it only for the creation of a directory.